Dataform/src/V1beta1/Client/DataformClient.php

```diff
@@ -1466,4 +1976,99 @@ public function listLocations(ListLocationsRequest $request, array $callOptions
     {
         return $this->startApiCall('ListLocations', $request, $callOptions);
     }
+
+    /**
+     * Gets the access control policy for a resource. Returns an empty policy
+    if the resource exists and does not have a policy set.
+     *
+     * The async variant is {@see DataformClient::getIamPolicyAsync()} .
+     *
+     * @example samples/V1beta1/DataformClient/get_iam_policy.php
+     *
+     * @param GetIamPolicyRequest $request     A request to house fields associated with the call.
+     * @param array               $callOptions {
+     *     Optional.
+     *
+     *     @type RetrySettings|array $retrySettings
+     *           Retry settings to use for this call. Can be a {@see RetrySettings} object, or an
+     *           associative array of retry settings parameters. See the documentation on
+     *           {@see RetrySettings} for example usage.
+     * }
+     *
+     * @return Policy
+     *
+     * @throws ApiException Thrown if the API call fails.
+     *
+     * @experimental
+     */
+    public function getIamPolicy(GetIamPolicyRequest $request, array $callOptions = []): Policy
+    {
+        return $this->startApiCall('GetIamPolicy', $request, $callOptions)->wait();
+    }
+
+    /**
+     * Sets the access control policy on the specified resource. Replaces
+    any existing policy.
+
+    Can return `NOT_FOUND`, `INVALID_ARGUMENT`, and `PERMISSION_DENIED`
+    errors.
```

Comment:
Improperly formatted?

Reply:
This looks like another protobuf bug (and maybe a new one?). We should file an issue. But since it's not causing any errors, it shouldn't block updates to the client libraries

Reply:
Reported here: https://github.com/protocolbuffers/protobuf/issues/14563